### ISSUE TITLE
Accessibility Office Hours - Slack comms image

### ISSUE THREAD

**Issue by chrisgrimley** [carbon-design-system/carbon#22459]
<img width="4096" height="1051" alt="Image" src="https://github.com/user-attachments/assets/364d871e-878e-4407-91db-456072e83e3c" />
<img width="4096" height="1051" alt="Image" src="https://github.com/user-attachments/assets/ab7eb4bb-da1c-46e9-b245-3cc1b0451e19" />
<img width="4096" height="1051" alt="Image" src="https://github.com/user-attachments/assets/a8f544d3-d51c-4ec2-93bb-1d8f8324239d" />
